ts/Shared/TimeBase.ts

```diff
@@ -929,7 +929,7 @@ namespace TimeBase {
 
     export interface DateTimeFormatOptions extends Intl.DateTimeFormatOptions {
         dateStyle?: 'full'|'long'|'medium'|'short';
-        fractionalSecondDigits?: number;
+        fractionalSecondDigits?: 1|2|3;
```

Comment:
How about 0?

Someone can use
`DateTimeFormat("en-US", { fractionalSecondDigits: 0 });` instaed of `.DateTimeFormat("en-US");` because of some reasons.

Reply:
More context: When I use Highcharts (in this case Grid) in a TypeScript environment, I get an error that the type extension is wrong because it tries to extend the default `1|2|3|undefined` with a `number`.

Reply:
Ok let's leave it like it is.